.azuredevops/deploy-platform-core-iam.yaml

```diff
@@ -0,0 +1,81 @@
+name: 1.0.$(BuildID)-${{ parameters.deployResources }}
```

Comment:
Why is this a separate pipeline?

Reply:
This is fine as discussed in the meeting.

Reply:
As discussed it is due to the dependency of app reg being created before we assign role